### DISCUSSION TITLE
Problem with Push Notification for mobile wallet app

### DISCUSSION THREAD

**Discussion by EvgenV** [WalletConnect/walletconnect-monorepo#4382] [Developer Support]
I am trying to register my wallet App for receiving push notifications from dApps. I am using WalletConnectSwiftV2 for integrating WalletConnection into the app. 
According to the documentation https://docs.walletconnect.com/web3wallet/push-notifications I call `register` method for sending deviceToken.

but logs contains following error
`HTTP error: {"errors":[{"message":"The provided Tenant ID, MY_PROJECT_ID, is invalid. Please ensure it's valid and the url is in the format /:tenant_id/...path","name":"tenant"}],"fields":[{"description":"Invalid Tenant ID, MY_PROJECT_ID","field":"tenant_id","location":"path"}],"status":"FAILURE"} code: 400, host: echo.walletconnect.com, path: /MY_PROJECT_ID/clients`
It seems that I've missed something during configuring my project, but can not figure out what exactly.

**Comment by glitch-txs** [comment]
hi could you open an issue in the swift repo?

**Reply by EvgenV** [reply]
Yes, sure. Thank you
